openshift-container-platform-4/policies/AC-Access_Control/component.yaml

```diff
@@ -115,149 +85,287 @@
+
+- control_key: AC-2 (8)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: not applicable
+  narrative:
+    - text: |-
+        This control is applicable to the identity service provider,
+        and outside the scope of OpenShift configuration.
 
 - control_key: AC-2 (9)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
-    - text: |
-        'This control is applicable to the identity service provider,
-        and outside the scope of OpenShift configuration.'
+    - text: |-
+        This control is applicable to the identity service provider
+        and outside the scope of CoreOS configuration.
 
 - control_key: AC-2 (10)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
-    - text: |
-        'OpenShift does not have the concept of group accounts.
+    - text: |-
+        OpenShift does not have the concept of group accounts.
 
         Should the organization allow shared credentials, the
         creation and management of such would fall under the scope
-        of operating the shared identity service and not OpenShift.'
+        of operating the shared identity service and not OpenShift.
 
 - control_key: AC-2 (11)
   standard_key: NIST-800-53
   covered_by: []
-  implementation_status: not applicable
+  implementation_status: partial
   narrative:
-    - text: |
-        'This control is applicable to the identity service provider,
-        and outside the scope of OpenShift configuration.'
+    - text: |-
+        This control will only be met as partial when OpenShift is
+        configured to connect to an Identity Provider (IdP).
+
+        More work is being done to address this control, Engineering
+        progress can be tracked via:
+
+        https://jira.coreos.com/browse/CMP-210
 
 - control_key: AC-2 (12)
   standard_key: NIST-800-53
   covered_by: []
   implementation_status: not applicable
   narrative:
     - key: a
-      text: |
-        'This control is applicable to the identity service provider,
-        and outside the scope of OpenShift configuration.'
+      text: |-
+        This control is applicable to the identity service provider,
+        and outside the scope of OpenShift configuration.
     - key: b
-      text: |
-        'This control is applicable to the identity service provider,
-        and outside the scope of OpenShift configuration.'
+      text: |-
+        This control is applicable to the identity service provider,
+        and outside the scope of OpenShift configuration.
 
 - control_key: AC-2 (13)
+  standard_key: NIST-800-53
+  covered_by: []
+  implementation_status: partial
+  narrative:
+    - text: |-
+        This control is applicable to the identity service provider,
+        and outside the scope of OpenShift configuration.
+
+        More work is being done to address this control, Engineering
+        progress can be tracked via:
```

Comment:
Could we clarify how/why this control is out of scope, but Engineering progress needs to be made?

Reply:
Fixed. But don't have anything further from Engineering.